lib/easy.c

```diff
@@ -194,6 +194,10 @@ static CURLcode global_init(long flags, bool memoryfuncs)
   }
 #endif
 
+#ifdef ENABLE_LIBPROXY
+  Curl_libproxy_global_init();
+#endif
```

Comment:
Provide the functions as empty macros if the feature is not built-in, then you don't need `#ifdefs` in the code like this.